modules/configuring-machine-pool-disk-volume-cli.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa_nodes/rosa-managing-worker-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="configuring_machine_pool_disk_volume_cli{context}"]
+= Configuring machine pool disk volume using the ROSA CLI
+
+.Prerequisite for day 1
+
+* You have the option to select the sizing for the default machine pool during cluster installation.
+
+.Procedure for day 1
+
+* Run the following command when creating your OpenShift cluster for the desired root disk size:
++
+[source,terminal]
+----
+$ rosa create cluster --worker-disk-size=<disk_size>
+----
+
+.Prerequisite for day 2
+
+* You have the option to select the sizing for the new day 2 machine pool after the cluster has been installed.
+
+.Procedure for day 2
+
+. Scale up the cluster by executing the following command:
++
+[source,terminal]
+----
+$ rosa create machinepool --cluster=<cluster_id> \ <1>
+                          --disk-size=<disk_size> \ <2>
```

Comment:
```suggestion
                          --disk-size=<disk_size> <2>
```

Reply:
Also, like mentioned in https://github.com/openshift/openshift-docs/pull/62116/files#r1302895585 a unit is required.